plugins/dependencies/src/main/kotlin/de/fayard/refreshVersions/internal/KotlinPoetry.kt

```diff
@@ -0,0 +1,103 @@
+package de.fayard.refreshVersions.internal
+
+import com.squareup.kotlinpoet.*
+
+
+internal data class Dependency(
+    val group: String = "",
+    val module: String = "",
+    val version: String = ""
+) {
```

Comment:
This can be confused with `Dependency` from Gradle. Do you have other name proposal that would fit for the use you have there?

Reply:
Renamed to `Library`